### DISCUSSION TITLE
How to undersample knee singlecoil_train k-space?

### DISCUSSION THREAD

**Discussion by juliadietlmeier** [facebookresearch/fastMRI#329] [General]
I am trying to build a simple reconstruction pipeline that is trained on fully-sampled and undersampled images. I have downloaded the NYU FastMRI knee singlecoil train data. I presume this is fully-sampled. How do I create the undersampled e.g. by factor x8 k-space? Thank you!

**Comment by mmuckley** [comment]
Hello @juliadietlmeier, the standard technique is by applying a mask, which simulates downsampling while ignoring some physics effects. We do this in quite a few areas of the repository for training models, mostly by creating a `mask_func` and incorporating it into the data transform pipeline.

https://github.com/facebookresearch/fastMRI/blob/edb5737d524c87db87dbe885e6de04e21bbbeeaf/fastmri_examples/varnet/train_varnet_demo.py#L27-L31
